### PR TITLE
docs lint-5

### PR DESCRIPTION
**Pull request by vinaykulk621**
This PR overs lint for `docs/api/en/lights`, `docs/api/en/lights/shadows`, `docs\api\en\loaders`, `docs\api\en\loaders\managers`, `docs\api\en\materials`, `docs\api\en\math` and `docs\api\en\math\interpolants` and some more files.
Ready to be merged.

### CODE REVIEW

File: docs/api/en/materials/MeshLambertMaterial.html [hunk 1]
```diff
@@ -14,101 +14,122 @@ <h1>[name]</h1>
...
-			scene.setAttribute( 'scrolling', 'no' );
-
-		}
+			if (/(iPad|iPhone|iPod)/g.test(navigator.userAgent)) {
+				const scene = document.getElementById("scene");
 
+				scene.style.width = getComputedStyle(scene).width;
+				scene.style.height = getComputedStyle(scene).height;
+				scene.setAttribute("scrolling", "no");
+			}
```
Comment (Mugen87) [inline docs/api/en/materials/MeshLambertMaterial.html:44]: I'm afraid this code is now formatted in the wrong code style.

I've also realized the code section in the geometry pages is now messed up. Can you please fix them?

File: docs/api/en/materials/MeshBasicMaterial.html [hunk 2]
```diff
@@ -12,15 +12,18 @@
...
 
+		<iframe
+			id="scene"
+			src="scenes/material-browser.html#MeshBasicMaterial"
+		></iframe>
 
-		<iframe id="scene" src="scenes/material-browser.html#MeshBasicMaterial"></iframe>
-
-		<script>
+<script>
```
Comment (Mugen87) [inline docs/api/en/materials/MeshBasicMaterial.html:26]: The formatting of this tag looks incorrect.

File: docs/api/zh/geometries/ShapeGeometry.html [hunk 3]
```diff
@@ -15,7 +15,7 @@ <h1>形状缓冲几何体（[name]）</h1>
 
 		<iframe id="scene" src="scenes/geometry-browser.html#ShapeGeometry"></iframe>
 
-		<script>
+				<script>
```
Comment (Mugen87) [inline docs/api/zh/geometries/ShapeGeometry.html:18]: Please undo this change.

In general, please do not touch translated pages.

Comment (vinaykulk621) [reply docs/api/zh/geometries/ShapeGeometry.html]: i have changed it that was a mistake.
